index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1 class="header">Hello Microverse</h1>
```

Comment:
- [ ] Please place this `<h1>` tag inside a `<header>` element. This is the proper HTML structure that is proposed by the project requirements, and your project will look more semantic.

For example:

```
  <header>
    <h1>Your text</h1>
  </header>
```